README.md

```diff
@@ -23,7 +23,7 @@ def largest?(array, value)
 end
 ```
 
-Snippet 2 - Big O:
+Snippet 2 - Big O: O(1) constant, not having to compare elements against each other
```

Comment:
This is actually O(n) because we have to visit every element in the customers array (`customers.each`). It's really O(n+n) because there are two sequential loops that are both O(n), but because they are sequential, we simplify to O(n) [we would have picked the worst case if the efficiencies were different, but they aren't in this case)